src/index.js

```diff
@@ -0,0 +1,50 @@
+import './style.css';
+import './style.scss';
+import 'bootstrap';
+
+const toDolist = document.getElementById('toDoList');
+const addBtn = document.getElementById('addBtn');
+const inputs = document.getElementsByClassName('forminput');
+let listArray = [];
+
```

Comment:
- [ ] You should have an array containing some tasks. Each task should have `description` (string), `completed` (boolean) and `index` (number) properties. Kindly add this.

  See example below
  ```
  const listArray = [
    {
      description: 'First task',
      completed: false,
      index: 1
    },
    {
      description: 'Second task',
      completed: true,
      index: 2
    },
    {
      description: 'Final task',
      completed: false,
      index: 3
    },
  ]
  ```